taichi/llvm/llvm_program.cpp

```diff
@@ -221,12 +219,6 @@ void LlvmProgramImpl::materialize_snode_tree(
   }
 }
 
-void LlvmProgramImpl::materialize_snode_expr_attributes(
-    SNodeGlobalVarExprMap &snode_to_glb_var_exprs_) {
-  for (auto &[snode, glb_var] : snode_to_glb_var_exprs_) {
```

Comment:
Hmm, we don't need to do this at all now..?

Reply:
As @ailzhang says, this is already done in
https://github.com/taichi-dev/taichi/blob/c0a5ebe070262cd691c6b71dd99127ac0bcea409/taichi/program/snode_expr_utils.cpp#L69
which is
https://github.com/taichi-dev/taichi/blob/c0a5ebe070262cd691c6b71dd99127ac0bcea409/taichi/ir/frontend_ir.h#L437-L440
Therefore it is redundant code.